vulcan/_lesson.py

```diff
@@ -61,6 +61,7 @@ class Lesson:
     time = ChildField(LessonTime, required=False)
     teacher = ChildField(Teacher, required=False)
     subject = ChildField(Subject, required=False)
+    changes = StringField(None, required=False)
```

Comment:
Please add here the key name `key="AdnotacjaOZmianie"` as above, and move this line below the `date` variable.